docs/topics/data-analysis/data-analysis-notebooks-output-formats.md

```diff
@@ -1,35 +1,230 @@
-[//]: # (title: Output formats supported by Kotlin Notebooks)
+[//]: # (title: Output formats supported by Kotlin Notebook)
 
-There will be examples for each output.
+Kotlin Notebook supports a variety of output types, including text, HTML, and images. With the help of external libraries,
+you can expand your output options and visualize your data with charts, spreadsheets, and more.
```

Comment:
Not sure but maybe it's worth mentioning how outputs work in Jupyter.
Each output is a JSON object mapping MIME types to some data, i.e.
```json
{
    "text/plain": "123"
}
```

Jupyter client (Kotlin Notebook plugin is also a client) picks  MIME type from this map that
- it supports
- has the highest priority among other types
and renders it in a way it can.
To use this mechanic directly, use this code:
```
MimeTypedResult(mapOf(
    "text/plain" to "123",
    //other mime:value pairs
))
```

I suggest letting know about this internals because it makes things more clear in general:
- text uses text/plain mime type
- buffered image serializes image into a base-64 string and uses image/png mime type. 
- Image class, as well as LATEX uses text/html mime type with the `img` tag in it
- User can use markdown as a cell output
<img width="377" alt="image" src="https://github.com/JetBrains/kotlin-web-site/assets/20319593/91fc3cfa-b942-4f6e-9320-88594c260b15">
- Dataframe and kandy use their own internal mime types, but they are backed by static HTML/images that allows them to be displayed on GitHub, i.e:
https://gist.github.com/ileasile/40f61d7a04490a5d9dad2314e7465bed